docs/OSUpgrades.md

```diff
@@ -14,83 +14,98 @@ These bootimages are built using [coreos-assembler](https://github.com/coreos/co
 Today, [the installer](https://github.com/openshift/installer/) pins the "bootimages"
 it uses, and released installers also pin the release image.  As noted above,
 release images contain `machine-os-content`, which can be a *different*
-RHCOS version.
+RHCOS version.  You can find the installer-pinned bootimage in e.g. [this file](https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json).
```

Comment:
maybe link to master instead of the release-4.4 branch

Reply:
agree

Reply:
But that file is highly likely to go away after https://github.com/openshift/enhancements/pull/201 - seems more useful to have an old but valid link.